src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -426,6 +426,18 @@ public AcquireTokenSilentParameterBuilder AcquireTokenSilent(IEnumerable<string>
                 loginHint);
         }
 
+        /// <summary>
```

Comment:
I'd probably add a remark that the tenant Id can be null if the authority type is ADFS.